content/javascript/concepts/arrays/terms/filter/filter.md

```diff
@@ -115,3 +115,23 @@ The output would be:
   team: "Red"
 }]
 ```
+
+## Codebyte Example
+
+The example below shows the `filter` array method being used with two criteria on the array `techCompany`.
+
+```codebyte/javascript
```

Comment:
Since this example is closely related to the above example, let's go ahead and remove the last example above  lines 77-117. Include in the sentence prior to the codebyte, that we are filtering out an array of objects.

Reply:
Hi, I have removed the required lines and changed the sentence above the codebyte example. I hope that it makes sense. Let me know if anything else needs changing.